organization/milestones.md

```diff
@@ -0,0 +1,53 @@
+---
+type: Resource
+---
+
+# Milestones
+
+We use milestones to plan our work around and set priorities. Milestones replace what other organisations would call 'projects' by focussing on the key achievements over certain deliverables.
+
+Milestones have an owner and can have a due date.
+
+Upcoming milestones:
+
+* Standard v0.2.0 2019-05-13 @bvhme
```

Comment:
This date has passed and we decided to go for 0.1.1 instead, this should be reassigned without a date.